### PR TITLE
Bugs/1149 test the add picture method

### PR DESCRIPTION
**Pull request by NAccuardi**
Fixes some of the tests.

### CODE REVIEW

File: EPPlusTest/DrawingTest.cs [hunk 1]
```diff
@@ -848,6 +848,458 @@ public void SaveCloseAndLoadDrawingsIntoWorkbook()
...
 		}
+
+		[TestMethod]
+		public void SaveCloseAndLoadDrawingsIntoWorkbookWithABMPImage()
+		{
+			string sheetName = "DrawingSheet";
+			FileInfo fileInfo = new FileInfo(base._worksheetPath + "SaveAndLoadTest.xlsx");
+			if (fileInfo.Exists)
+				fileInfo.Delete();
+			using (var package = new ExcelPackage())
```
Comment (emdelaney) [inline EPPlusTest/DrawingTest.cs]: Let's wrap everything from here on down in a try/finally block so we can delete the file in the finally block.

File: EPPlusTest/DrawingTest.cs [hunk 2]
```diff
@@ -848,6 +848,458 @@ public void SaveCloseAndLoadDrawingsIntoWorkbook()
...
+		}
+
+		[TestMethod]
+		public void SaveCloseAndLoadDrawingsIntoWorkbookWithAGIFImage()
+		{
+			string sheetName = "DrawingSheet";
+			FileInfo fileInfo = new FileInfo(base._worksheetPath + "SaveAndLoadTest.xlsx");
+			if (fileInfo.Exists)
+				fileInfo.Delete();
+			using (var package = new ExcelPackage())
```
Comment (emdelaney) [inline EPPlusTest/DrawingTest.cs]: Same here, let's try/finally it up.

File: EPPlusTest/DrawingTest.cs [hunk 3]
```diff
@@ -848,6 +848,458 @@ public void SaveCloseAndLoadDrawingsIntoWorkbook()
...
+		}
+
+		[TestMethod]
+		public void SaveCloseAndLoadDrawingsIntoWorkbookWithAJPEGImage()
+		{
+			string sheetName = "DrawingSheet";
+			FileInfo fileInfo = new FileInfo(base._worksheetPath + "SaveAndLoadTest.xlsx");
+			if (fileInfo.Exists)
+				fileInfo.Delete();
+			using (var package = new ExcelPackage())
```
Comment (emdelaney) [inline EPPlusTest/DrawingTest.cs]: One more! You get the idea.

File: EPPlusTest/DrawingTest.cs [hunk 4]
```diff
@@ -848,6 +848,458 @@ public void SaveCloseAndLoadDrawingsIntoWorkbook()
...
+			finally
+			{
+				if (fileInfo.Exists)
+					fileInfo.Delete();
+			}
+		}
+		#endregion
+
+		#region AddPicture Tests
+		// Test adding from an Image object (with and without URI).
```
Comment (emdelaney) [inline EPPlusTest/DrawingTest.cs]: Are these tests still coming? Or can we remove this region?

Comment (NAccuardi) [inline EPPlusTest/DrawingTest.cs]: We can get rid of that section. That was there when I made this branch.

File: EPPlusTest/DrawingTest.cs (hunk 1 above)
Comment (NAccuardi) [inline EPPlusTest/DrawingTest.cs]: Easy Peasy.

Comment (NAccuardi) [inline EPPlusTest/DrawingTest.cs]: ✔️

File: EPPlusTest/DrawingTest.cs (hunk 2 above)
Comment (NAccuardi) [inline EPPlusTest/DrawingTest.cs]: ✔️

File: EPPlusTest/DrawingTest.cs (hunk 3 above)
Comment (NAccuardi) [inline EPPlusTest/DrawingTest.cs]: ✔️

File: EPPlusTest/DrawingTest.cs (hunk 4 above)
Comment (NAccuardi) [inline EPPlusTest/DrawingTest.cs]: ✔️

**Pull request merged**